### DISCUSSION TITLE
Wrong "Insufficient space on disk for database" error message

### DISCUSSION THREAD

**Discussion by xX-MrN0b0dy-Xx** [MobiusSync/MobiusSync#80] [Q&A] [answered]
First of all, let me say that this app is awesome, it’s unique in all the App Store. I absolutely love it and I’ll be glad to write a 5 star review if you can solve this problem that I’m having right now.
I’m trying to sync a folder with files of any kind and any dimension for a total of ~70 GB (originally 207 GB but reduced with ignore patterns) on an iPad with 201 GB free space.
In the beginning I’ve set the minimum space to 1% (both in device default and in folder sync settings) but when the folder became about ~58 GB an error appeared that said: “insufficient space on disk for database” (see attached image for more details). This is very strange because there were ~143GB of unused space… I tried to change that 1% to 1GB (both in device default and in folder sync settings) and the sync started working again until only 2.8GB were left and the same error appeared (~67GB unused space at that time). So I changed it to 100MB but it didn’t work and after I set it to 1MB and it worked… 
I don’t understand why this problem appears, there is plenty of space… 
I thought that could be a problem with maximum dimension allowed for an app (when you check the storage info in iPad settings, all the folders and the files synced with Möbius are detected as “Documents & Data” of Möbius but such limits doesn’t exist. I used to have an app that was 112GB included “Documents & Data”… 
The problem is not the device because I have similar problems with my iPhone that sync the same folder and has much less unused space (~20GB now that the folder is synced) and it even happens if I uninstall it and reinstall. What’s fun is that on my iPhone, after the first time I’ve seen the error and changed 1% with 1GB everything worked smoothly without further problems.
![IMG_0776](https://user-images.githubusercontent.com/112854121/188515797-e9d46801-6635-4849-bff5-0f13a14a8fd2.PNG)

**Comment by xX-MrN0b0dy-Xx** [comment]
any update?

**Reply by wealthychef1** [reply]
I also am having this problem

**Comment by valsaven** [comment]
Same

**Comment by SimonPickup** [comment, marked as answer]
Please see #17
